src/4.x/upgrading.md

```diff
@@ -47,6 +47,21 @@ ELASTICSEARCH_USER=
 ELASTICSEARCH_PASSWORD=
 ```
 
+You will also have to replace `ELASTICSEARCH_PREFIX` with `SCOUT_PREFIX`:
+
+```diff
+-  ELASTICSEARCH_PREFIX="your_prefix_here"
++  SCOUT_PREFIX="your_prefix_here"
+```
+
+### `providers.php` changes
+
+You need to add the Scout Elasticsearch provider to your providers file:
+
+```php
+Matchish\ScoutElasticSearch\ElasticSearchServiceProvider::class,
+```
```

Comment:
This can be removed with https://github.com/rapidez/rapidez/pull/96#discussion_r2088956840